apps/trading/components/market-header/market-header.tsx

```diff
@@ -25,7 +26,14 @@ export const MarketHeader = () => {
           onChange={setOpen}
           trigger={
             <HeaderTitle>
-              {data.tradableInstrument.instrument.code}
+              <span>
+                {data.tradableInstrument.instrument.code}
+                <MarketProductPill
```

Comment:
suggestion(non blocking): we should really be using a data grid component here. Instead can we use the following. I think this looks nicer than the pill

```
<span className="ml-2 text-xs uppercase text-muted">
{ProductTypeShortName[
  data.tradableInstrument.instrument.product.__typename
  ]}
</span>
```